python/frontend/cirrus/examples/test.py

```diff
@@ -0,0 +1,29 @@
+import time
```

Comment:
Shouldn't this be called somewhere, or am I missing something?

Reply:
The time module is used in line 9 to keep track of how long the timeout is.

Reply:
I meant this test.py file. You added it but is it called anywhere?

Reply:
Still waiting for answer.

Reply:
The test.py file is imported into an iPython notebook and the function within is called when you want to test the accuracy to check if it's increasing. See logistic_regression.ipynb.